Nio/Matrix Wrapper/NIORoom.swift

```diff
@@ -95,6 +95,22 @@ class NIORoom: ObservableObject {
         }
     }
 
+    func sendImage(image: UIImage) {
+        guard let imageData = image.jpeg(.lowest) else { return }
+        //swiftlint:disable:next redundant_optional_initialization
+        var localEcho: MXEvent? = nil
+        // TODO: Use localEcho to show sent message until it actually comes back
+        room.sendImage(
+            data: imageData,
+            size: image.size,
+            mimeType: "image/jpeg",
+            thumbnail: image,
+            localEcho: &localEcho
+        ) {_ in
```

Comment:
Very minor nitpick 😅 
```suggestion
        ) { _ in
```